go.mod

```diff
@@ -1,12 +1,18 @@
 module github.com/ActiveState/langtools
 
-go 1.12
+go 1.22
 
 require (
-	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
-	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
 	github.com/ericlagergren/decimal v0.0.0-20191206042408-88212e6cfca9
 	github.com/stretchr/testify v1.4.0
 	golang.org/x/text v0.3.2
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 )
+
+require (
+	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
+	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
+	github.com/davecgh/go-spew v1.1.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	gopkg.in/yaml.v2 v2.2.2 // indirect
+)
```

Comment:
Is tat correct? There are now two `require(..)` blocks?

Reply:
I think `go mod` separates direct and indirect dependencies into distinct blocks. This is the first time I've seen it though.